upass/__init__.py

```diff
@@ -52,6 +53,14 @@
 __all__ = ('config',)
 
 
+# Parse CLI args
+parser = argparse.ArgumentParser(
+        description='upass: a TUI frontend for pass.')
+parser.add_argument(
+        '-s', '--search', help='Start in search mode',
+        action='store_true')
+args = vars(parser.parse_args())
```

Comment:
I don't like `vars` here, just access it as `args.search`.

Reply:
Done